UKHO.PeriodicOutputService/UKHO.PeriodicOutputService.API.FunctionalTests/Helpers/GetBatchStatus.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace UKHO.PeriodicOutputService.API.FunctionalTests.Helpers
+{
+    public class GetBatchStatus
+    {
+        static HttpClient httpClient = new HttpClient();
```

Comment:
```suggestion
        private static readonly HttpClient httpClient = new();
```
Can we do this change same as done in GetCatalogue.cs

Reply:
Resolved